ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml

```diff
@@ -282,10 +282,29 @@ tests:
 - as: vsphere-ipi-customized-resource-f28
   cron: 56 22 18 * *
   steps:
-    cluster_profile: vsphere-connected-2
+    cluster_profile: vsphere-elastic
     test:
     - ref: cucushift-installer-check-cluster-health
+    - ref: cucushift-installer-check-vsphere-disk
```

Comment:
I think we can directly add this step to `cucushift-installer-rehearse-vsphere-ipi-customized-resource` provision chain.

Reply:
ok